.github/workflows/sync-wiki.yml

```diff
@@ -0,0 +1,65 @@
+name: Sync wiki
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'wiki/**'
+      - 'README.md'
+
+jobs:
+  sync:
+    if: github.repository != github.event.repository.full_name
```

Comment:
Should be `if: github.repository == github.event.repository.full_name` not `!=`.